storage/memory/storage_test.go

```diff
@@ -41,7 +41,7 @@ func (s *StorageSuite) TestObjectStorageSetAndGet(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(h.String(), Equals, "dcf5b16e76cce7425d0beaef62d79a7d10fce1f5")
 
-	e, err := os.Get(h)
+	e, err := os.Get(h, core.AnyObject)
```

Comment:
we should test that this particular hash is found if you ask for Any or Commit and it is not found when you ask for the rest object types.